policies/appgateway_diagnostics/policy.json

```diff
@@ -0,0 +1,89 @@
+{
+    "properties": {
+        "displayName": "HMCTS Application Gateway Diagnostics Policy",
+        "policyType": "Custom",
+        "mode": "All",
+        "parameters": {
+            "eventHubAuthRule": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Event Hub Authorization Rule Id",
+                    "description": null
+                },
+                "allowedValues": [
+                    "/subscriptions/2307d175-7e49-434b-9ac2-515529b845f2/resourceGroups/soc-core-infra-sbox-rg/providers/Microsoft.EventHub/namespaces/soc-sbox-eventhubns/authorizationRules/RootManageSharedAccessKey"
```

Comment:
don't you need the prod one here too?

Reply:
ticket stipulates testing in sbox first then changing to prod

Reply:
this is just the policy, assignments dictate where it's applied too.

you should be able to test it in it's current state now anyway as this PR will have created a policy automatically